core/api.go

```diff
@@ -8,22 +8,25 @@ import (
 	"mime/multipart"
 	"net/http"
 	"net/url"
-	"regexp"
 	"strings"
 	"time"
 
-	"github.com/gin-contrib/cors"
+	"github.com/textileio/textile-go/pb"
+
+	"github.com/golang/protobuf/proto"
+
 	limit "github.com/gin-contrib/size"
 	"github.com/gin-gonic/gin"
 	"github.com/golang/protobuf/jsonpb"
-	"github.com/golang/protobuf/proto"
+	cors "github.com/rs/cors/wrapper/gin"
```

Comment:
swapping in a difference cors lib. this one is better maintained, simpler to use, and supports wildcard usage which makes cors on localhost easier/better